python/llm/dev/benchmark/all-in-one/run-pvc.sh

```diff
@@ -0,0 +1,7 @@
+source /opt/intel/oneapi/setvars.sh
```

Comment:
run max

Reply:
maybe `run-max-gpu.sh` is better (as we also have Xeon Max)

Reply:
Done